base/include/Command.h

```diff
@@ -166,4 +167,29 @@ class StepCommand : public Command
 	}
 
 
+};
+
+class ValvePassThroughCommand : public Command
+{
+public:
+	ValvePassThroughCommand() : Command(Command::CommandType::ValvePassThrough)
+	{
+	}
+
+	size_t getSerializeSize()
+	{
+		return Command::getSerializeSize() + sizeof(numOfFrames) + 128;
```

Comment:
that 128 may not be needed given that we are already padding it by 1024 byes in the base class